CHANGELOG.md

```diff
@@ -3,11 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [2.0.0] - 2019-02-26
 
 ### Added
-- Migrate from Vault to AWS Secrets Manager - see [#6](https://github.com/ExpediaInc/apiary-authorization/issues/6).
-  This is is backward incompatible change, adds new variables ldap_ca_cert, ldap_secret_name, check README for usage.
+- Migrate from Vault to AWS Secrets Manager - see [#6](https://github.com/ExpediaInc/apiary-authorization/issues/6).  
+This is backward incompatible change, adds new variables `ldap_ca_cert`, `ldap_secret_name`, please refer to the [README.md](README.md) for usage.
```

Comment:
It's still unclear to me what exactly the backwards incompatible change is - i.e. if I move to this new version what exactly do I need to do?

Reply:
In previous versions, module was using Vault to manage secrets and ssl certificates.
With this version, secrets are automatically generated and saved in SecretsManager, so any datalake deployments need to be updated to use these new secrets, 
and SSL certificates need to be added manually to IAM, since we removed that functionality from module.

Reply:
OK, so aside from the vault/aws secrets manager change the user will need to provide values for 'ldap_ca_cert' (for the SSL stuff) and 'ldap_secret_name' (for ldap creds). Which I guess is all covered in the README that this refers to.